onboarding/mutations.py

```diff
@@ -32,6 +34,17 @@ def mutate(root, info, **validated_data):
         )
         return CreateBusiness(business=business, created=created)
 
+# class UploadMutation(graphene.Mutation):
```

Comment:
could you add a comment if we want to save this commented out code? Else remove it.